src/tscore/unit_tests/test_ArgParser.cc

```diff
@@ -79,10 +79,10 @@ TEST_CASE("Parsing test", "[parse]")
   parsed_data = parser.parse(argv2);
   REQUIRE(parsed_data.get("init") == true);
   REQUIRE(parsed_data.get("subinitopt") == true);
-  REQUIRE(parsed_data.get("globaly") == true);
-  REQUIRE(parsed_data.get("globaly").size() == 2);
-  REQUIRE(parsed_data.get("globaly")[0] == "y1");
-  REQUIRE(parsed_data.get("globaly")[1] == "y2");
+  REQUIRE(parsed_data.get("globally") == true);
+  REQUIRE(parsed_data.get("globally").size() == 2);
+  REQUIRE(parsed_data.get("globally")[0] == "y1");
+  REQUIRE(parsed_data.get("globally")[1] == "y2");
```

Comment:
I think "globaly" is not a spelling mistake.

Reply:
globally means relating to the whole thing, and is in english dictionaries.
globaly isn't in english dictionaries -- what does it mean? relating to a glob? 
(or is your point that it needs fixing in more spots?)

Reply:
It's actually Global Y. There are also globalx and globalz.

Reply:
Yeah. It is a test for the argument parser. So the test defines a few string like "global x", "global y" and "global z" as tokens for the parser. So it is not a spelling mistake for "globaly". Rather it is not referring to one of the tokens to be found inside the string.